source/_integrations/zha.markdown

```diff
@@ -33,41 +33,45 @@ There is currently support for the following device types within Home Assistant:
 - Sensor
 - Switch
 
-## ZHA exception and deviation handling
-
-Zigbee devices that deviate from or do not fully conform to the standard specifications set by the [Zigbee Alliance](https://zigbeealliance.org) may require the development of custom [ZHA Device Handlers](https://github.com/dmulcahey/zha-device-handlers) (ZHA custom quirks handler implementation) to for all their functions to work properly with the ZHA integration in Home Assistant. These ZHA Device Handlers for Home Assistant can thus be used to parse custom messages to and from Zigbee devices.
+## Compatible hardware
 
-The custom quirks implementations for zigpy implemented as ZHA Device Handlers for Home Assistant are a similar concept to that of [Hub-connected Device Handlers for the SmartThings Classics platform](https://docs.smartthings.com/en/latest/device-type-developers-guide/) as well as that of [Zigbee-Shepherd Converters as used by Zigbee2mqtt](https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html), meaning they are each virtual representations of a physical device that expose additional functionality that is not provided out-of-the-box by the existing integration between these platforms.
+ZHA integration support using one of the multiple available Zigbee coordinator radio modules that is compatible with [zigpy](https://github.com/zigpy/zigpy). zigpy is an open source Python library implementing a hardware independent Zigbee stack that uses a modular design with separate Python libraries for Zigbee coordinator radio modules from different manufacturers, with each library capable of interfacing with USB and GPIO radio hardware adapters/modules over different UART serial communication protocols. Support for radio libraries in this ZHA integration include; [zigpy-deconz](https://github.com/zigpy/zigpy-deconz) for Dresden-Elektronik deCONZ based radios, [bellows](https://github.com/zigpy/bellows) for Silicon Labs EmberZNet/EZSP based radios, and [zigpy-xbee](https://github.com/zigpy/zigpy-xbee) for XBee based Zigbee radios. There are currently also experimental support for a couple additional radio libraries; [zigpy-cc](https://github.com/sanyatuning/zigpy-cc) for Texas Instruments Z-Stack based radios, and [zigpy-zigate](https://github.com/doudz/zigpy-zigate) for ZiGate based radios.
 
-## Known working Zigbee radio modules
+### Known working Zigbee radio modules
 
-- dresden elektronik deCONZ based Zigbee radios (via the [zigpy-deconz](https://github.com/zigpy/zigpy-deconz) library for zigpy)
+- **Dresden Elektronik deCONZ based Zigbee radios**
```

Comment:
"dresden elektronik" is written in lower case. Please respect the writing style of the brand.
```suggestion
- **dresden elektronik deCONZ based Zigbee radios**
```